### PR TITLE
Set migrations up for Maximum group assignment members #511

### PR DESCRIPTION
**Pull request by dinever**
This is the migration needed for #511. 

These changes add the `max_members` column onto `group_assignments` for setting a maximum number of members in group assignment.

### CODE REVIEW

File: db/migrate/20160321131514_add_max_members_to_group_assignments.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddMaxMembersToGroupAssignments < ActiveRecord::Migration
+  def change
+    add_column :group_assignments, :max_members, :integer
+  end
+end
```
Comment (johndbritton) [inline db/migrate/20160321131514_add_max_members_to_group_assignments.rb]: You need a newline at the end of this file

**Comment by johndbritton** [comment]
:+1:

File: db/migrate/20160321131514_add_max_members_to_group_assignments.rb (hunk 1 above)
Comment (dinever) [inline db/migrate/20160321131514_add_max_members_to_group_assignments.rb]: @johndbritton 
Fixed. Thanks!

**Comment by johndbritton** [comment]
Migrated

```ruby
  ActiveRecord::SchemaMigration Load (4.5ms)  SELECT "schema_migrations".* FROM "schema_migrations"
Migrating to AddMaxMembersToGroupAssignments (20160321131514)
   (1.7ms)  BEGIN
== 20160321131514 AddMaxMembersToGroupAssignments: migrating ==================
-- add_column(:group_assignments, :max_members, :integer)
   (13.2ms)  ALTER TABLE "group_assignments" ADD "max_members" integer
   -> 0.0141s
== 20160321131514 AddMaxMembersToGroupAssignments: migrated (0.0144s) =========

  SQL (1.8ms)  INSERT INTO "schema_migrations" ("version") VALUES ($1)  [["version", "20160321131514"]]
   (3.5ms)  COMMIT
```

File: db/migrate/20160321131514_add_max_members_to_group_assignments.rb (hunk 1 above)
Comment (johndbritton) [inline db/migrate/20160321131514_add_max_members_to_group_assignments.rb]: cc/ https://github.com/education/classroom/issues/514

**Pull request merged**